docs/roadmap.md

```diff
@@ -1,21 +1,19 @@
-# Windows UI Library Roadmap
+# Windows UI Library Overview
 
-WinUI is the modern native UI platform for Windows. This document outlines the prospective roadmap for WinUI through 2021. WinUI is under active development by the Windows developer platform team and the roadmap will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here aren't exhaustive or guaranteed. We welcome your feedback on the roadmap: please feel free to contribute to existing issues or [file a new issue](https://github.com/microsoft/microsoft-ui-xaml/issues).
+WinUI is the modern native UI platform for Windows with two active generations:
 
-**Note: For a detailed look at the target availability dates for certain features in WinUI 3, see [our feature roadmap](#winui-3-feature-roadmap).**
+1. **WinUI 3**: The latest, 3rd generation of WinUI that ships the entire WinUI stack decoupled from the operating system as a part of the [Windows App SDK](https://docs.microsoft.com/windows/apps/windows-app-sdk/).
 
-WinUI continues to advance and improve across both generations of the technology: 
+2. **WinUI 2**: The previous generation of the WinUI stack for UWP apps, consisting of a XAML and Visual Layer built directly into the Windows 10 operating system, and a controls library built on top of the OS, delivered via NuGet, and hosted at this repository. WinUI 2 will continue to be supported with bug, reliability, and security fixes.
```

Comment:
Out of curiosity, is UWP XAML part of WinUI "Gen" 2 or does WinUI "Gen" 2 only mean the library?

Also, what is WinUI Gen 1? @ryandemopoulos didn't you say something about that in a community call?

Reply:
We don't really talk about Gen 1, but it's a good question.  :)  Before WinUI 2 went public, there was an earlier internal-only version that we used to build a few apps in Microsoft.  In retrospect we probably should have launched that as 1.0, but since we already had a bunch of customers on "WinUI 1", we decided to move to "WinUI 2" when we went public.

In the bucket of "things I'd change if I had a time machine".  :)